apps/dotlottie-viewer/src/pages/Perf.tsx

```diff
@@ -0,0 +1,342 @@
+import { Player as LottieWeb } from '@lottiefiles/react-lottie-player';
+import { isMobile } from 'react-device-detect';
+import { setCanvasKit, Skottie as SkottiePlayer } from '../components/skottie';
+import skottieWasmUrl from '../../node_modules/canvaskit-wasm/bin/full/canvaskit.wasm?url';
+import InitCanvasKit from 'canvaskit-wasm/bin/full/canvaskit';
+import dotLottieWasmUrl from '../../../../packages/web/src/core/dotlottie-player.wasm?url';
+
+setDotLottieWasmUrl(window.location.origin + dotLottieWasmUrl);
+
+const animations = [
+  '1643-exploding-star.json',
+  '5317-fireworkds.json',
+  '5344-honey-sack-hud.json',
+  '11555.json',
+  '27746-joypixels-partying-face-emoji-animation.json',
+  'a_mountain.json',
+  'abstract_circle.json',
+  'alien.json',
+  'anubis.json',
+  'balloons_with_string.json',
+  'batman.json',
+  'birth_stone_logo.json',
+  'calculator.json',
+  'card_hover.json',
+  'cat_loader.json',
+  'coin.json',
+  'confetti.json',
+  'confetti2.json',
+  'confettiBird.json',
+  'customer.json',
+  'dancing_book.json',
+  'dancing_star.json',
+  'day_to_night.json',
+  'dodecahedron.json',
+  'down.json',
+  'dropball.json',
+  'duck.json',
+  'earth_hour.json',
+  'emoji_enjoying.json',
+  'emoji.json',
+  'error_404.json',
+  'fiery_skull.json',
+  'fleche.json',
+  'flipping_page.json',
+  'fly_in_beaker.json',
+  'focal_test.json',
+  'foodrating.json',
+  'frog_vr.json',
+  'fun_animation.json',
+  'funky_chicken.json',
+  'game_finished.json',
+  'geometric.json',
+  'glow_loading.json',
+  'gradient_background.json',
+  'gradient_infinite.json',
+  'gradient_sleepy_loader.json',
+  'gradient_smoke.json',
+  'growup.json',
+  'guitar.json',
+  'hamburger.json',
+  'happy_holidays.json',
+  'happy_trio.json',
+  'hola.json',
+  'holdanimation.json',
+  'hourglass.json',
+  'insta_camera.json',
+  'isometric.json',
+  'kote.json',
+  'la_communaute.json',
+  'like_button.json',
+  'like.json',
+  'loading_rectangle.json',
+  'lolo_walk.json',
+  'lolo.json',
+  'loveface_emoji.json',
+  'masking.json',
+  'material_wave_loading.json',
+  'merging_shapes.json',
+  'message.json',
+  'morphing_anim.json',
+  'open_envelope.json',
+  'personal_character.json',
+  'polystar_anim.json',
+  'polystar.json',
+  'property_market.json',
+  'radar.json',
+  'ripple_loading_animation.json',
+  'rufo.json',
+  'sad_emoji.json',
+  'sample.json',
+  'seawalk.json',
+  'skullboy.json',
+  'snail.json',
+  'starburst.json',
+  'starstrips.json',
+  'starts_transparent.json',
+  'stroke_dash.json',
+  'swinging.json',
+  'text.json',
+  'text_anim.json',
+  'text2.json',
+  'threads.json',
+  'train.json',
+  'uk_flag.json',
+  'voice_recognition.json',
+  'walker.json',
+  'water_filling.json',
+  'waves.json',
+  'woman.json',
+  'yarn_loading.json',
+];
+
+const urlPrefix = 'https://raw.githubusercontent.com/thorvg/thorvg/main/examples/resources/lottie/';
```

Comment:
One thing to note is that we occasionally update the sample resources in the folder. You might want to copy them to your own repo to prevent breakage from ThorVG updates.